### DISCUSSION TITLE
第6节课作业(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#441] [作业]
请在下方提交作业链接,格式为
-->报名序号+连接:
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by lishuang70345** [comment]
451+https://blog.csdn.net/weixin_42748604/article/details/135683401

**Comment by admlrz** [comment]
https://blog.csdn.net/weixin_44581312/article/details/135758050

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec6

**Comment by MIracleyin** [comment]
https://nnkwjqimr7.feishu.cn/docx/BlnOd3u4soGLcgxVZkZcnHAtnnf?from=from_copylink

**Comment by dearSocrates** [comment]
https://blog.csdn.net/rabbit9798/article/details/135784249

**Comment by cemcke** [comment]
http://t.csdnimg.cn/FPMr4

**Comment by Tian-CASIA** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135854284
